app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -111,6 +116,50 @@ class ChatActivity : AppCompatActivity(), IChatView {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
So the minimum version needed to use this feature is Lollipop.

Reply:
but we don't use lollipop